up42/tasking.py

```diff
@@ -155,5 +155,31 @@ def get_quotations(
             )
         return self._query_paginated_output(url)
 
+    def decide_quotation(self, quotation_id: str, desicion: str) -> dict:
+        """Accept or reject a quotation for a tasking order.
+        This operation is only allowed on quotations with the NOT_DECIDED status.
+
+        Args:
+            quotation_id (str): The target quotation ID.
+            desicion (str): The decision made for this quotation.
+
+        Returns:
+            dict: The confirmation to the decided quotation plus metadata.
+        """
+        if desicion not in ["ACCEPTED", "REJECTED"]:
```

Comment:
I am taking a look at [API reference](https://docs.up42.com/developers/api#tag/tasking/operation/listTaskingQuotation), the endpoint actually doesn't need `decision` value.
Why are we making it compulsory here?